docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+tracker:
+  image: ${HERON_IMAGE}
+  command: heron-tracker
+  ports:
+    - "8888:8888"
+  volumes:
+    - "herondata:/root/.herondata"
+ui:
+  image: ${HERON_IMAGE}
+  command: heron-ui --tracker_url=http://192.168.33.11:8888
+  ports:
+    - "8889:8889"
```

Comment:
As a heads up [chef-zero](https://github.com/chef/chef-zero#valid-options) also runs on this port by default and may cause conflicts.